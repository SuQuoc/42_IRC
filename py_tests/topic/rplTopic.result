:AfterLife 001 client0 :Welcome to the Internet Relay Network, user0
:AfterLife 005 client0 NICKLEN=9 CHANTYPES=#& CHANMODES=oiklt :are supported by this server
:AfterLife 001 client1 :Welcome to the Internet Relay Network, user1
:AfterLife 005 client1 NICKLEN=9 CHANTYPES=#& CHANMODES=oiklt :are supported by this server
:client0!user0@10.12.2.6 JOIN #chan * :user0
:client1!user1@10.12.2.6 JOIN #chan * :user1
:client1!user1@10.12.2.6 JOIN #chan * :user1
#chan :No topic is set
:client0!user0@10.12.2.6 TOPIC #chan :new topic
:client0!user0@10.12.2.6 TOPIC #chan :new topic
#chan :new topic
:client0!user0@10.12.2.6 QUIT :Leaving
