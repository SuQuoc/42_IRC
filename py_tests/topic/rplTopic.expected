:AfterLife 001 client0 :Welcome to the Internet Relay Network, user0
:AfterLife 001 client1 :Welcome to the Internet Relay Network, user1
:client0!user0@10.14.3.10 JOIN #chan * :user0
#chan :No topic is set
:client0!user0@10.14.3.10 TOPIC #chan :new topic
#chan :new topic
